requirements/edx/paver.txt

```diff
@@ -5,3 +5,4 @@ lazy==1.1
 path.py==7.2
 watchdog==0.7.1
 python-memcached
+libsass==0.10.0
```

Comment:
This is a set of python bindings for libsass. Not a port, but just a thin layer over the C++ lib.